ReleaseChecklist.md

```diff
@@ -102,12 +102,15 @@ At least a day before the release:
 ### Post-release
  - [ ] Make sure the documentation for the new release has been published successfully.
        Go to the [documentation status page at ReadTheDocs](https://readthedocs.org/projects/solidity/) and verify that the new version is listed, works and is marked as default.
- - [ ] Remove "still in progress" warning from the release notes.
- - [ ] Publish the blog posts.
+ - [ ] Remove "still in progress" warning from the [release notes](https://github.com/ethereum/solidity/releases).
+ - [ ] Merge the [blog posts](https://github.com/ethereum/solidity-blog/pulls) related to the release.
  - [ ] Create a commit to increase the version number on ``develop`` in ``CMakeLists.txt`` and add a new skeleton changelog entry.
- - [ ] Announce on Twitter, including links to the release and the blog post.
-       Use ``#xp`` at the end of the tweet to automatically cross post the announcement to Fosstodon.
+ - [ ] Update the release information section on [soliditylang.org](https://github.com/ethereum/solidity-portal](https://github.com/ethereum/solidity-portal/blob/master/index.html#L22).
```

Comment:
```suggestion
 - [ ] Update the release information section [in the source of soliditylang.org](https://github.com/ethereum/solidity-portal/blob/master/index.html).
```

By they way, linking to a specific line is convenient but risky. It could easily move after some changes in the page structure and the link would become misleading without us noticing. I think that just linking to the whole page is clear enough.